rules/cloud/gcp/gcp_firewall_rule_modified_or_deleted.yml

```diff
@@ -0,0 +1,25 @@
+title: Google Cloud Firewall Modified or Deleted
+id: fe513c69-734c-4d4a-8548-ac5f609be82b
+description: Detects  when a firewall rule is modified or deleted in Google Cloud Platform (GCP).
+author: Austin Songer
+status: experimental
+date: 2021/08/13
+references:
+    - https://cloud.google.com/kubernetes-engine/docs/how-to/audit-logging
+    - https://developers.google.com/resources/api-libraries/documentation/compute/v1/java/latest/com/google/api/services/compute/Compute.Firewalls.html
+logsource:
+  service: gcp.audit
+detection:
+    selection:
+        gcp.audit.method_name: 
+            - v*.compute.firewalls.delete
+            - v*.compute.firewalls.patch
+            - v*.compute.firewalls.update
+    condition: selection
```

Comment:
`Compute.Firewalls.Insert` sould be a very low event in the dayly works (or you have a very bad organization).